cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java

```diff
@@ -834,13 +835,14 @@ public Tracing getTracing() {
     }
 
     /**
-     * Creates the ClusterRoleBinding which is used to bind the Kafka Connect SA to the ClusterRole
-     * which permissions the Kafka init container to access K8S nodes (necessary for rack-awareness).
+     * Creates a ClusterRoleBinding which is used to bind the SA to the strimzi-kafka-client ClusterRole.
+     *
+     * @param name Name of the cluster role binding.
      *
      * @return The cluster role binding.
      */
-    public ClusterRoleBinding generateClusterRoleBinding() {
-        if (rack == null) {
+    public ClusterRoleBinding generateKafkaClientClusterRoleBinding(String name) {
```

Comment:
I think this is weird. I understand that you probably want to share the method but use two different names. But I'm not sure if it should be part of the public API. Maybe you can have protected method for the CRB with the name parameter and public ones in the model classes without the name which would use their own name to call the protected methods.

Basing the existence of the resource on the name seems also completely counterintuitive. It should continue use the `rack` field.

I also don't follow why this needed the renaming. Seems like unnecessary change. The old Javadoc seemed to be also more descriptive.